### PR TITLE
Fix typo when loading auto-arpa config

### PR DESCRIPTION
**Pull request by kkzo**
Resolved a typo that prevented the auto-arpa configuration options from loading.

### CODE REVIEW

**Review by ross — APPROVED** [review]
Doh. Should have checked this before I moved forward with 1.1.0 in https://github.com/octodns/octodns/pull/1062. Will queue up a patch release once this is merged.

File: tests/config/simple-arpa.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 manager:
   max_workers: 2
   auto_arpa:
+    populate_should_replace: True
```
Comment (ross) [inline tests/config/simple-arpa.yaml:4]: Any specific reasons for adding this and checking it in the tests. It seems fine just wondering what lead to the change as it doesn't seem to be related to coverage. Just testing that the param was passed through correctly?

Comment (kkzo) [reply tests/config/simple-arpa.yaml:4]: Thanks for the quick release. Yes, I added that to test if the param is actually making its way through correctly.
